addons/account/models/account.py

```diff
@@ -149,6 +149,12 @@ def _validate_tags(self):
         if self.tag_ids and (len(neg_tags) !=1 or len(pos_tags) != 1):
             raise ValidationError(_("If tags are defined for a tax report line, only two are allowed on it: a positive and a negative one."))
 
+    @api.constrains('tag_name')
```

Comment:
```suggestion
    @api.constrains('tag_name', 'country_id')
```
So we can prevent this when someone changes Country on existing record. (IDK know the actual workflow but just saying based on code.)

Reply:
country_id isn't supposed to change, but I agree adding it to the constraint is more consistent. Thank you for your comment !